### PR TITLE
[T3A3][T13-A4] Sean Zhang

### PR DESCRIPTION
**Pull request by puzzledsean**
Implemented Address into multiple objects, updated expected and input txt for testing

Referenced #411 and #424 for input/expected txt for testing (I used roughly the same tests, slightly modified)

### CODE REVIEW

**Comment by puzzledsean** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Block.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	public String block;
```
Comment (zhongwei-z) [inline src/seedu/addressbook/data/person/Block.java]: If you have a getter for it, need you set it to `public`?

File: src/seedu/addressbook/data/person/Block.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	
+	public Block(String block){
+		this.block = block;
+	}
+	
+	public String getBlock(){
+		return this.block;
+	}
+}
```
Comment (zhongwei-z) [inline src/seedu/addressbook/data/person/Block.java]: Can one change one's block number?

**Comment by zhongwei-z** [comment]
Good work. Think about whether there is a need for a getter for each new class.

**Pull request closed without merging**